production/helm/loki/templates/read/statefulset-read.yaml

```
@@ -141,7 +141,7 @@ spec:
       {{- end }}
       {{- with .Values.read.dnsConfig }}
       dnsConfig:
-        {{- tpl . $ | nindent 8 }}
+        {{- toYaml . | nindent 8 }}
```

This should prevent a breaking change:

```suggestion
        {{- if kindIs "map" . }}
        {{- tpl (toYaml .) $ | nindent 8 }}
        {{- else }}
        {{- tpl . $ | nindent 8 }}
        {{- end }}
```